src/components/MovieBooking/MovieModal.jsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { useParams } from "react-router-dom";
+import {
+  ModelStyle,
+  ImageStyles,
+  ModelDataFlex,
+  HeadingStyle,
+  SubHeadingStyle,
+  ModelScreenStyle,
+  CloseButton,
+  SeatHeading,
+  SeatHeadingStyle,
+} from "../Styles/Modal.styles";
+import { imgUrl } from "../ constants/global";
+import CloseIcon from "../Assets/closePng.png";
+const MovieModal = ({ propsData }) => {
```

Comment:
```suggestion
const MovieModal = ({ selectSeatArray, selectSeatFun, visibleFun }) => {
```
props is already an object that bundles all the passed values together. So please do not pass the props into another object.